jenkins/debian/debos/scripts/stretch-ltp.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Important: This script is run under QEMU
+
+set -e
+
+# Build-depends needed to build the test suites, they'll be removed later
+BUILD_DEPS="
+            acl \
+            acl-dev \
+            attr \
+            autoconf \
+            automake \
+            ca-certificates \
+            gawk \
+            git \
+            gcc \
+            libaio1 \
+            libc6-dev \
+            libcap2 \
+            make \
+            openssl \
+            zip \
+"
+
+apt-get install --no-install-recommends -y  ${BUILD_DEPS}
+
+BUILDFILE=/build_info.txt
+echo '  "tests_suites": [' >> $BUILDFILE
+
+# Build and install IO performance suites
+########################################################################
+TMP_TEST_DIR="/tmp/tests/clone"
+mkdir -p ${TMP_TEST_DIR} && cd ${TMP_TEST_DIR}
+
+TESTDIR=ltp
+TEST_REPO=https://github.com/linux-test-project/ltp.git
+
+git clone --depth=1 -b 20190115 ${TEST_REPO}
+cd ${TESTDIR}
+
+echo '    {"name": "${TESTDIR}", "git_url": "${TEST_REPO}", "git_commit": ' \"`git rev-parse HEAD`\" '}' >> $BUILDFILE
+
+make autotools
+./configure --prefix=/opt/ltp
+make V=1
+make install V=1
```

Comment:
If you look at what the other test suites do, they strip the binaries and remove unnecessary files for this particular use-case.  The size of the rootfs image is particularly important as it will be downloaded many times, and when running as a ramdisk it'll obviously take up some RAM.

So I think there should be a few more steps there to do the same thing, with the only difference that the binaries will be installed in `/opt` instead of `/usr`.  Again, it would be good to see the difference in the ramdisk size as a result of doing this - but even if it's small, every little helps.